dashboard/config/course_offerings/applab-intro.json

```diff
@@ -5,7 +5,11 @@
   "is_featured": false,
   "assignable": true,
   "curriculum_type": null,
-  "marketing_initiative": "HOC",
```

Comment:
It looks like you are losing some settings here. Did you seed your local environment before serializing?

Reply:
Thank you! My mistake was I seeded from my branch, and not from staging first then switching over. The new commit fixes this!